pyproject.toml

```diff
@@ -1,15 +1,92 @@
-[tool.black]
+[build-system]
+requires = ["flit_core >=3.4,<4"]
+build-backend = "flit_core.buildapi"
+
+[project]
+name = "arviz"
+readme = "README.md"
+requires-python = ">=3.11"
+license = {file = "LICENSE"}
+authors = [
+    {name = "ArviZ team", email = "arvizdevs@gmail.com"}
+]
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Intended Audience :: Developers",
+    "Intended Audience :: Science/Research",
+    "Intended Audience :: Education",
+    "License :: OSI Approved :: Apache Software License",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: 3.13",
+]
+dynamic = ["version", "description"]
+dependencies = [
+  "arviz_base >=0.7.0",
+  "arviz_stats[xarray] >=0.7.0",
+  "arviz_plots >=0.7.0",
+]
+
+[tool.flit.module]
+name = "arviz"
+
+[project.urls]
+source = "https://github.com/arviz-devs/arviz"
+tracker = "https://github.com/arviz-devs/arviz/issues"
+documentation = "https://python.arviz.org"
+funding = "https://opencollective.com/arviz"
+
+[project.optional-dependencies]
+# user facing optional blocks
+netcdf4 = ["arviz_base[netcdf4]"]
+h5netcdf = ["arviz_base[h5netcdf]"]
+zarr = ["arviz_base[zarr]"]
+matplotlib = ["arviz_plots[matplotlib]"]
+bokeh = ["arviz_plots[bokeh]"]
+plotly = ["arviz_plots[plotly]"]
+
+# dev related blocks
+doc = [
+    "h5netcdf",
+    "jupyter-sphinx",
+    "matplotlib",
+    "myst-parser[linkify]",
+    "myst-nb",
+    "pydata-sphinx-theme>=0.13",
+    "sphinx>=5",
+    "sphinx-copybutton",
+    "sphinx-design",
+    "sphinx-notfound-page",
+    "sphinxcontrib-youtube",
+]
+test = ["pytest"]
+check = ["build", "pre-commit"]
```

Comment:
Are these used for testing/checking purposes? I don't see them in the GA files... Does tox use them under the hood? Or, are those for developers to do:

```shell
pip install -e .[test]
```

?